Prod Build fails

```
docker compose -f docker-compose.prod.yml build
 => [frontend dev 3/4] COPY . .                                                                                                                                                    0.0s
 => ERROR [frontend dev 4/4] RUN npm install                                                                                                                                       6.1s
------
 > [frontend dev 4/4] RUN npm install:                                                                                                                                                  
5.880 npm error code ERESOLVE
5.880 npm error ERESOLVE could not resolve                                                                                                                                              
5.880 npm error
5.882 npm error While resolving: csaf-provider-scan-frontend@0.1.0
5.882 npm error Found: @vitejs/plugin-vue@6.0.5
5.882 npm error node_modules/@vitejs/plugin-vue
5.882 npm error   dev @vitejs/plugin-vue@"^5.0.0" from the root project
5.882 npm error
5.882 npm error Could not resolve dependency:
5.882 npm error dev @vitejs/plugin-vue@"^5.0.0" from the root project
5.882 npm error
5.882 npm error Conflicting peer dependency: vite@5.4.21
5.882 npm error node_modules/vite
5.882 npm error   peer vite@"^5.0.0 || ^6.0.0" from @vitejs/plugin-vue@5.2.4
5.882 npm error   node_modules/@vitejs/plugin-vue
5.882 npm error     dev @vitejs/plugin-vue@"^5.0.0" from the root project
5.882 npm error
5.882 npm error Fix the upstream dependency conflict, or retry this command with --force or --legacy-peer-deps to accept an incorrect (and potentially broken) dependency resolution.
5.883 npm error
5.883 npm error
5.883 npm error For a full report see:
5.883 npm error /root/.npm/_logs/2026-04-23T19_52_01_694Z-eresolve-report.txt
5.889 npm notice
5.889 npm notice New minor version of npm available! 11.12.1 -> 11.13.0
5.889 npm notice Changelog: https://github.com/npm/cli/releases/tag/v11.13.0
5.889 npm notice To update run: npm install -g npm@11.13.0
5.889 npm notice
5.890 npm error A complete log of this run can be found in: /root/.npm/_logs/2026-04-23T19_52_01_694Z-debug-0.log
------
[+] build 0/2
 ⠙ Image csaf-provider-scan-backend  Building                                                                                                                                       6.5s
 ⠙ Image csaf-provider-scan-frontend Building                                                                                                                                       6.5s
Dockerfile:8

--------------------

   6 |     COPY . .

   7 |     

   8 | >>> RUN npm install

   9 |     

  10 |     EXPOSE 8080

--------------------

target frontend: failed to solve: process "/bin/sh -c npm install" did not complete successfully: exit code: 1
```

--- a/.env
+++ b/.env
@@ -2,6 +2,8 @@
 PORT_BACKEND=48090
 PORT_FRONTEND=48091
 PORT_VALIDATOR=48092
+# in production environments, set this port to 443
+#VITE_BACKEND_PORT=$PORT_BACKEND
 
 ## Scan Slots
 SCAN_SLOTS=10

--- a/docker-compose.prod.yml
+++ b/docker-compose.prod.yml
@@ -31,7 +31,7 @@ services:
       target: prod
       args:
         - VITE_FOOTER_TEXT=${FOOTER_TEXT:-}
-        - VITE_BACKEND_PORT=443
+        - VITE_BACKEND_PORT=${VITE_BACKEND_PORT:-48090}
     container_name: frontend
     ports:
       - "${PORT_FRONTEND:-48091}:8080"

--- a/docker-compose.prod.yml
+++ b/docker-compose.prod.yml
@@ -50,7 +50,7 @@ services:
   validator:
     build:
       context: ./validator
-      target: dev
+      target: prod
     container_name: validator
     ports:
       - "${PORT_VALIDATOR:-48092}:8082"

--- a/validator/Dockerfile
+++ b/validator/Dockerfile
@@ -5,22 +5,24 @@ WORKDIR /app
 ## Installs
 
 RUN npm install @secvisogram/csaf-validator-service
-RUN npm ci
 
 # Setup
 EXPOSE 8082
 
 COPY ./command.sh ./command.sh
-RUN chmod -x ./command.sh
+RUN chmod +x ./command.sh
 
 FROM build AS dev
 
 ## Command
 CMD ["sh", "/app/command.sh"]
 
-# FIXME
-FROM nginx:1.29.4 AS prod
+FROM node:22.21-alpine AS prod
 
-COPY --from=build /app/dist/client/browser /usr/share/nginx/html
+WORKDIR /app
 
-CMD ["sh", "/app/command.sh"]
+COPY --from=build /app/node_modules /app/node_modules
+
+EXPOSE 8082
+
+CMD ["sh", "-c", "cd /app/node_modules/@secvisogram/csaf-validator-service && npm start"]

